Dockerfile

```diff
@@ -18,7 +19,7 @@ RUN apt-get update && \
     mkdir /public && \
     groupadd -g $GID pretalxuser && \
     useradd -r -u $UID -g pretalxuser -d /pretalx -ms /bin/bash pretalxuser && \
-    echo 'pretalxuser ALL=(ALL) NOPASSWD:SETENV: /usr/bin/supervisord' >> /etc/sudoers
+    echo 'pretalxuser ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
I am not sure, but removing the `SETENV` will break the system, since environment variables will not be exported to the sudo target, but we need them to do that from the pretalx script to go into the gunicorn.

Reply:
I've added a change to _deployment/docker/pretalx.bash_, to use `sudo -E` when running our web app, to reserve environment variables.

Reply:
Thanks, that should do it.

But a question: Why the change at all, it is not related to `less` at all?

Reply:
We need to view logs (with `less`) written by `supervisord`, but the files are under `root` user, hence need `sudo`.